core/src/main/scala/kafka/tools/DumpLogSegments.scala

```diff
@@ -638,7 +638,7 @@ object DumpLogSegments {
       " Instead, the value-decoder-class option can be used if a custom RLMM implementation is configured.")
     private val shareStateOpt = parser.accepts("share-group-state-decoder", "If set, log data will be parsed as share group state data from the " +
       "__share_group_state topic.")
-    private val skipRecordMetadataOpt = parser.accepts("skip-record-metadata", "Whether to skip printing metadata for each record.")
+    private val skipRecordMetadataOpt = parser.accepts("skip-record-metadata", "Whether to skip printing control records and metadata for each other record.")
```

Comment:
What does "other" mean?

Reply:
```suggestion
    private val skipRecordMetadataOpt = parser.accepts("skip-record-metadata", "Whether to skip printing `control records`, and metadata for each other record.")
```
might highlight this as well. I mistaken it for control(s), records, and metadata, but control records is on term.

Reply:
@Yunyung we only have control and metadata records on this topic, so other means metadata record.